plugins/main/public/components/common/welcome/components/sca_scan/sca_scan.tsx

```diff
@@ -220,7 +220,7 @@ export const ScaScan = compose(
               </RedirectAppLinks>
             </EuiFlexItem>
             <EuiFlexItem grow={false} style={{ marginTop: 12 }}>
-              <EuiBadge color='secondary'>{lastScan.policy_id}</EuiBadge>
+              <EuiBadge color='secondary'>{lastScan?.policy_id}</EuiBadge>
```

Comment:
**thought**: This solves the problem if the `lastScan` variable is undefined or something like that, but for this cause, the text could appear as `undefined`. I don't think we want this on the UI.